iotedgedev/deploymentmanifest.py

```diff
@@ -256,5 +230,49 @@ def _validate_createOptions(self):
         if (validation_success):
             self.output.info("Validation for all createOptions passed.")
         else:
-            self.output.info("Errors found during createOptions validation. Please check the logs for details.")
+            self.output.warning("Errors found during createOptions validation. Please check the logs for details.")
+        return validation_success
+
+    def _validate_createOptions_lengh(self, module_name, module_info):
```

Comment:
Consider making the _validate_create method names consistent.  Recommend renaming this to `_validate_create_options_length`

Reply:
updated